docs/reference/ImageDraw.rst

```diff
@@ -255,7 +255,7 @@ Methods
 
     Draw a shape.
 
-.. py:method:: PIL.ImageDraw.ImageDraw.text(xy, text, fill=None, font=None, anchor=None, spacing=0, align="left", direction=None, features=None)
+.. py:method:: PIL.ImageDraw.ImageDraw.text(xy, text, fill=None, font=None, anchor=None, spacing=0, align="left", direction=None, language=None, features=None)
```

Comment:
Please add the new `language` parameter to the end (to this and other methods).

Some people may already be calling them with positional rather than named arguments, and we don't want to break the API for them.

Reply:
fixed! thanks for the input.

Reply:
You're welcome! Still a few to do: please put `language` after `features` here.